mifosng-android/src/main/java/com/mifos/api/datamanager/DataManagerClient.java

```diff
@@ -316,10 +317,10 @@ public DataManagerClient(BaseApiManager baseApiManager,
      *
      * @param clientId          Client Id
      * @param identifierPayload IdentifierPayload
-     * @return GenericResponse
+     * @return IdentifierCreationResponse
      */
-    public Observable<GenericResponse> createClientIdentifier(int clientId,
-                                                              IdentifierPayload identifierPayload) {
+    public Observable<IdentifierCreationResponse> createClientIdentifier(
```

Comment:
@droidchef  Will it be a better approach to check if the `GenericResponse` got `resourceId` and similar values that we want to check and then proceed accordingly? You once told about the naming and readability. I created the new model following those lines.